bundles/org.eclipse.swt/Readme.Win32.md

```diff
@@ -6,29 +6,24 @@ The SWT windows port is based on the Win32 api.
 
 The natives are compiled via an ant build.xml script with the Microsoft Visual Studio (2019) C++ compiler.
 
-# Development overview
-SWT is made up of 2 parts, Java and the natives (written in C).
-
-* **Java Part**  
-If you only make changes in the Java part or need to configure SWT for running snippets,
-then you don't need to recompile the natives, instead, you can just use the pre-compiled
-libraries in the binary repository.
-
-* **Natives Part**  
-(Make sure the binary project for your platform is imported and open in your workspace).
-
 ## Configure workspace to run snippets.
 
 * Download & install Java *JDK* (Last tested on JDK9) http://www.oracle.com/technetwork/java/javase/downloads/index.html
 * Download and install Eclipse. Either the "Eclipse IDE for Eclipse Committers" or a recent integration build:
   https://download.eclipse.org/eclipse/downloads/
 * (Optionally) install CDT from marketplace if you want to work on C/Native parts of SWT.
-* (if not already installed) installed EGit integration.
-* Open the git perspective. Add the following two repositories, which can be found on: (use git protocol)
-	- https://github.com/eclipse-platform/eclipse.platform.swt
-* From swt repo, import most projects (except Cocoa/Gtk, root 'swt' project, once)
-* And import the project that reflects your platform.
-* In the snippet project, add the swt project as dependency to launch snippets.
+* Install EGit.
```

Comment:
Since you have already solved this in a different way, but in general would have been simpler if we would just write that one should import `Eclipse projects` at specific locations, which would already exclude for example the `binaries` project which is just Maven project.